console.rst

```diff
@@ -142,6 +142,11 @@ attribute. For example, you may want a command to create a user::
         }
     }
 
+.. versionadded:: 7.3
+
+    Invokable commands that don't extend the base ``Command`` class were
```

Comment:
```suggestion
    Support for invokable commands that don't extend the base ``Command`` class was
```

Like this?

Reply:
Much better. I just updated this. Thanks!